jutut/settings.py

```diff
@@ -52,6 +52,7 @@
     'bootstrapform',
     'bootstrapform_jinja',
     'django_filters',
+    'plotly',
```

Comment:
I think this is not needed. I have not checked, but plotly shouldn't have django app.